pytest_spec/patch.py

```diff
@@ -54,6 +59,18 @@ def _print_class_information(self):
     self._first_triggered = True
 
 
+def _remove_class_prefix(nodeid):
+    return re.sub("^Test", "", nodeid)
+
+
+def _split_words(nodeid):
```

Comment:
I would rename this to `_split_capital_letters` to be a smidge more clear??? Maybe split words is better...